src/net/sourceforge/kolmafia/request/FightRequest.java

```diff
@@ -8129,6 +8133,21 @@ private static boolean handleCosmicBowlingBall(String text) {
     return false;
   }
 
+  private static boolean handleTrainset(String text) {
+    if (!text.contains("Your toy train moves ahead to the")) {
```

Comment:
The messages for some reason seem to have their html stripped out, the station names are in bold.

I didn't really care to look into it as the message itself should be unique. Although, it could be more unique.

Reply:
You can pass in the full html to handleTrainset if you want